scripts/zones/The_Garden_of_RuHmet/mobs/Ix_aern_drg.lua

```diff
@@ -56,20 +56,12 @@ function onMobDeath(mob, player, isKiller)
     DespawnMob(wynavA);
     DespawnMob(wynavB);
     DespawnMob(wynavC);
-
-    -- Pick a new for PH Ix'Aern (DRG)
-    SetServerVariable("[SEA]IxAernDRG_PH", AwAernDRGGroups[math.random(1, #AwAernDRGGroups)] + math.random(0, 2));
 end;
 
 -----------------------------------
 -- OnMobDespawn
 -----------------------------------
 function onMobDespawn( mob )
-    -- Despawn pets.
-    DespawnMob(wynavA);
```

Comment:
The onMobDespawn willl always run after death, so that one could be left and the duplicate code in oMobDeath taken out but then there's a delay before the helpers will begin their depop (though that might be retail, I dunno).